### PR TITLE
Multi-tool bug with dropping files

### PR DESCRIPTION
**Pull request by dhenry437**
This should fix a bug I found where the PDF Multi-tool filename input box would not update and remain disabled when dragging a file in but not when using the filepicker.

# License Agreement for Contributions
By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under MPL 2.0 (Mozilla Public License Version 2.0) license. 

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

### CODE REVIEW

**Comment by sbplat** [comment]
This change doesn't seem to be working on my end. I'm using Edge with version 120.0.2210.133.

**Comment by sbplat** [comment]
If everything is still functional then this LGTM.

File: src/main/resources/static/js/multitool/fileInput.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import PdfContainer from "./PdfContainer";
```
Comment (sbplat) [inline src/main/resources/static/js/multitool/fileInput.js]: ESM requires a file extension to be provided.

File: src/main/resources/static/js/multitool/fileInput.js [hunk 2]
```diff
@@ -43,10 +45,14 @@ const addFileDragListener = (callback) => {
...
             //maybe
         }).finally(() => {
+            // Hide and remove the overlay
             if (overlay) {
                 overlay.remove();
                 overlay = null;
             }
+
+            // Enable/disable filename input
+            PdfContainer.updateFilenameInput()
```
Comment (sbplat) [inline src/main/resources/static/js/multitool/fileInput.js]: `updateFilenameInput` isn't a static class method, so this wouldn't work.

File: src/main/resources/static/js/multitool/PdfContainer.js [hunk 3]
```diff
@@ -74,32 +74,35 @@ class PdfContainer {
...
+            filenameInput.value = "";
+            this.filename = null;
+            downloadBtn.disabled = true;
+        } else {
+            this.filename = filenameInput.value;
         }
 
-        input.click();
+        if (this.filename === null || this.filename === undefined) {
+            filenameInput.value = files[0].name;
```
Comment (sbplat) [inline src/main/resources/static/js/multitool/PdfContainer.js]: I don't think `files` is defined here?

**Comment by sbplat** [comment]
I am not exactly sure how it was working on your end. The comments I just left above highlight some potential issues that could break functionality. It would be beneficial to test it and ensure everything related to the change still works as expected before proceeding with a PR.

Also, in the future, could you please create an issue relating to this bug first? Thanks!

**Comment by dhenry437** [comment]
So from what I can see you fixed-up all my poor code? Thankyou. I'll take all your points into consideration in future.

**Review by sbplat — APPROVED** [review]

**Review by Frooodle — APPROVED** [review]

**Pull request merged**